var/spack/repos/builtin/packages/paraview/get_version_without_git.patch

```diff
@@ -0,0 +1,29 @@
+--- ParaView-v5.4.1.orig/CMake/ParaViewDetermineVersion.cmake	2017-10-12 14:36:02.000000000 -0600
++++ ParaView-v5.4.1/CMake/ParaViewDetermineVersion.cmake	2017-10-12 14:39:39.000000000 -0600
+@@ -36,16 +36,16 @@
+     # this function.
+     return ()
+   elseif (NOT PARAVIEW_GIT_DESCRIBE)
```

Comment:
Possibly, but I don't think that is the correct way to handle this because `PARAVIEW_GIT_DESCRIBE` is used in other parts of Paraview such as Catalyst and that could have side effects I am unaware of, so that would be more of a hack than I am doing here in my opinion.

Reply:
Patch are brittle and needed to port forward all the time, hence if you have an option to get away without a patch, but simply overwriting a cmake variable I would really prefer that.

Reply:
Have a look at https://github.com/Kitware/ParaView/blob/master/CMake/ParaViewDetermineVersion.cmake, setting `PARAVIEW_GIT_DESCRIBE` to an empty string will do exactly the same as not running git (what your patch is doing.)

Reply:
But Catalyst sets `PARAVIEW_GIT_DESCRIBE`. If I start trying to force that variable to be blank, I don't know what the side effects of that are.

https://github.com/Kitware/ParaView/blob/master/Catalyst/catalyze.py#L284

Reply:
I see, so go with the idea below.